docs/compiler.md

```diff
@@ -66,5 +66,5 @@ By using the expected compiler, the expected libraries are correctly linked.
 
 Pyccel offers the `--conda-warnings` flag which takes one of the following options : `off`, `basic`, `verbose`.
 This controls the visibility of Conda-related warnings. The default value is `basic` which indicates when folders are ignored.
-The `verbose` option additionally outputs a list of all ignored folders. These folders can be un-ignored using a user-defined compiler file as described above.
+The `verbose` option additionally outputs a list of all ignored folders. If these folders souldn't be ignored then this can be avoided by using a user-defined compiler file as described above.
```

Comment:
Typo: souldn't -> shouldn't (maybe "should not" is better)

Reply:
Fixed 2ecc2f18